scripts/components/advanced-color-picker/advanced-color-picker.js

```diff
@@ -118,6 +121,7 @@ export const AdvancedColorPicker = (props) => {
 
 	return (
 		<BaseControl
+			className={additionalClasses ?? ''}
```

Comment:
Just a note for the future that these kinds of changes are the reason for the minor version update.

Semver is: `Major.Minor.Security`, and adding any kind of feature, no matter how small (until it's a BC break), goes in the minor release.